packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -47,12 +57,14 @@ function readBlobSection(node: NodeTypes) {
             // "id": <node name>
             // "data": <blob>
             blobs.set(blob.getString(1), blob.getBlob(3).arrayBuffer);
+            fastBlobPath += 1;
             continue;
         }
 
         /**
          * More generalized workflow
          */
+        slowBlobPath += 1;
```

Comment:
I believe (please validate) that we could control these flows and thus really all we care is counting number of times we take slow path. If we took it too many times (let's say above 100), then it's a problem. If we can argue that even once taking it is bad and we should fix (I think we can argue that), then we can change check as appropriately. I'd still do counting (instead of having error event here) and report later summary, as logging error many times is slow.

Reply:
I now added it to trees latest telemetry.